package.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euxo pipefail
+
+git clone https://github.com/fraunhoferfokus/deckschrubber.git /tmp/deckschrubber
+cd /tmp/deckschrubber
+git checkout $DECKSCHRUBBER_TAG
```

Comment:
You can combine this into one by doing `git clone --branch $DECKSCHRUBBER_TAG ...`